core/src/main/java/com/google/instrumentation/stats/RpcConstants.java

```diff
@@ -40,86 +40,117 @@
   // RPC client {@link MeasurementDescriptor}s.
   public static final MeasurementDescriptor RPC_CLIENT_ERROR_COUNT =
       MeasurementDescriptor.create(
-          "/rpc/client/error_count",
+          "/grpc.io/client/error_count",
           "RPC Errors",
           MeasurementUnit.create(0, scalar));
   public static final MeasurementDescriptor RPC_CLIENT_REQUEST_BYTES =
       MeasurementDescriptor.create(
-          "/rpc/client/request_bytes",
+          "/grpc.io/client/request_bytes",
           "Request bytes",
           MeasurementUnit.create(0, bytes));
   public static final MeasurementDescriptor RPC_CLIENT_RESPONSE_BYTES =
       MeasurementDescriptor.create(
-          "/rpc/client/response_bytes",
+          "/grpc.io/client/response_bytes",
           "Response bytes",
           MeasurementUnit.create(0, bytes));
   public static final MeasurementDescriptor RPC_CLIENT_ROUNDTRIP_LATENCY =
       MeasurementDescriptor.create(
-          "/rpc/client/roundtrip_latency",
+          "/grpc.io/client/roundtrip_latency",
           "RPC roundtrip latency msec",
           MeasurementUnit.create(-3, seconds));
   public static final MeasurementDescriptor RPC_CLIENT_SERVER_ELAPSED_TIME =
       MeasurementDescriptor.create(
-          "/rpc/client/server_elapsed_time",
+          "/grpc.io/client/server_elapsed_time",
           "Server elapsed time in msecs",
           MeasurementUnit.create(-3, seconds));
   public static final MeasurementDescriptor RPC_CLIENT_UNCOMPRESSED_REQUEST_BYTES =
       MeasurementDescriptor.create(
-          "/rpc/client/uncompressed_request_bytes",
+          "/grpc.io/client/uncompressed_request_bytes",
           "Uncompressed Request bytes",
           MeasurementUnit.create(0, bytes));
   public static final MeasurementDescriptor RPC_CLIENT_UNCOMPRESSED_RESPONSE_BYTES =
       MeasurementDescriptor.create(
-          "/rpc/client/uncompressed_response_bytes",
+          "/grpc.io/client/uncompressed_response_bytes",
           "Uncompressed Response bytes",
           MeasurementUnit.create(0, bytes));
+  public static final MeasurementDescriptor RPC_CLIENT_STARTED_COUNT =
+      MeasurementDescriptor.create(
+          "/grpc.io/client/started_count",
+          "Number of client RPCs started",
+          MeasurementUnit.create(0, scalar));
+  public static final MeasurementDescriptor RPC_CLIENT_FINISHED_COUNT =
+      MeasurementDescriptor.create(
+          "/grpc.io/client/finished_count",
+          "Number of client RPCs finished",
+          MeasurementUnit.create(0, scalar));
+  public static final MeasurementDescriptor RPC_CLIENT_REQUEST_COUNT =
+      MeasurementDescriptor.create(
+          "/grpc.io/client/request_count",
+          "Number of client RPC requests",
```

Comment:
A quote from the original definition in the bug:

> Unlike the above four (started and finished count) that count RPCs (streams), these four (request & response count) count messages

I'll update the description here.